ctre/basemotorcontroller.py

```diff
@@ -114,9 +142,9 @@ def set(self, mode: ControlMode, demand0: float, demand1: float = 0.0):
                 work |= int(demand0) & 0xFF
             else:
                 work = int(demand0)
-            self.setDemand(self.sendMode, work, 0)
-        elif self.controlMode in [ControlMode.Velocity, ControlMode.Position, ControlMode.MotionMagic, ControlMode.MotionMagicArc, ControlMode.MotionProfile]:
-            self.setDemand(self.sendMode, int(demand0), 0)
+            self._set_4(self.sendMode, demand0, demand1, demand1Type)
```

Comment:
This should be `work` instead of `demand0`.

Reply:
Good catch.